src/router/routes/ssoUsers.js

```diff
@@ -145,13 +145,17 @@ router.put(
         process.env.EMAIL_INVITATION_JWT_SECRET
       );
 
-      if (tokenData.email === verifyBody.email && tokenData.code === verifyBody.code) {
-        // Assingn SSO user to group:
+      if (tokenData.code !== verifyBody.code) {
+        logger.info('- Invitation code not matching');
+        return res.status(400).json('Invalid invitation link');
```

Comment:
res.status(400).send()  
`invalid invitation link` doesn't look like valid json

Reply:
unless you were going for something like `{message: 'Invalid invitation link'}`